core/src/main/resources/reference.conf

```diff
@@ -43,6 +43,19 @@ submissionmonitor {
   trackDetailedSubmissionMetrics = true
 }
 
+data-source {
+  coordinated-access {
```

Comment:
OOC, just like Cromwell, do we need to update rawls.conf config in firecloud-develop for this change?

Reply:
Nope. I turned the coordination on by default on the next couple lines below.

The new knob allows us to turn off the functionality, if needed for some reason, without needing to recompile the code.